web/modules/custom/dpl_related_content/templates/dpl-related-content-slider.html.twig

```diff
@@ -0,0 +1,5 @@
+{{ title }}
+
+{% for item in items %}
+  {{ item }}
+{% endfor %}
```

Comment:
Consider if this template is necessary.

If it is a dummy template then could we just delete it?

Reply:
It is necessary; If the template doesnt exist, Drupal loses its mind.
But yeah, this markup will never be displayed - it is overwritten by `novel`